### PR TITLE
add metadata alias in TransactionResult

### PR DESCRIPTION
**Pull request by mukulljangid**
fixes https://github.com/XRPLF/xrpl4j/issues/239

### CODE REVIEW

**Review by sappenin — CHANGES_REQUESTED** [review]
I think this will work, but let's add some unit test coverage to validate the fix.

File: xrpl4j-integration-tests/src/test/java/org/xrpl/xrpl4j/tests/LedgerResultIT.java [hunk 1]
```diff
@@ -70,4 +72,26 @@ void getClosedLedgerResult() throws JsonRpcClientErrorException {
...
     assertThat(ledgerResult.ledger().parentCloseTime()).isNotEmpty();
   }
+
+  @Test
+  void getLedgerResult() throws JsonRpcClientErrorException {
+
+    final XrplClient mainnetClient = new MainnetEnvironment().getXrplClient();
+    final LedgerResult ledgerResult = mainnetClient.ledger(LedgerRequestParams.builder()
+      .ledgerSpecifier(LedgerSpecifier.VALIDATED)
+        .transactions(true)
```
Comment (nkramer44) [inline xrpl4j-integration-tests/src/test/java/org/xrpl/xrpl4j/tests/LedgerResultIT.java:82]: can you reformat these lines so they are lined up?

**Review by nkramer44 — CHANGES_REQUESTED** [review]
Just one small nit then this is good!

**Review by sappenin — APPROVED** [review]
LGTM!

**Review by nkramer44 — APPROVED** [review]

**Pull request merged**